### PR TITLE
Add a resource for open source access

### LINKED ISSUE #36: Document guidance about access to OSS projects for financial services
### Description of Problem:

In #26 "access to OSS projects (when are maintainers vs. when contributors)" came up as a common challenge faced by financial services institutions.

### Potential Solutions:

Add a new resource page to the [OSR microsite](https://osr.finos.org) to provide some guidance on this topic.

### PR DESCRIPTION
**Pull request by vmbrasseur**
Closes #36

### CODE REVIEW

File: docs/operations/resources.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ sidebar_label: The Role of Operations
 - [FOSS Policy](foss-policy.md)
 - [Considerations and Guidance for Electronic Communications Policies and Open Source](electronic-communications.md)
 - [Considerations and Guidance for Records Management regards Open Source](records-management.md)
+- [Working with DLP Limitations to Open Source Access](./project-access.md)
```
Comment (psmulovics) [inline docs/operations/resources.md:13]: Do we need the ./ ?

**Review by psmulovics — APPROVED** [review]

**Comment by vmbrasseur** [comment]
Suggestions from #48:

* For the exception: is this a one-time access or required for a certain period of time? If the latter, how will it be managed?
* Granularity of license/rights of the project

Other comments/todos:

* Clean up the bullet points at the end of the document

**Comment by vmbrasseur** [comment]
Changes made per suggestions above. Ready for another (maybe the final?) review, team.

**Review by psmulovics — APPROVED** [review]
Looks great!

**Comment by vmbrasseur** [comment]
Thanks, Peter! Merging now. 👍

**Pull request merged**